src/test/java/com/rultor/agents/twitter/TweetsTest.java

```diff
@@ -46,6 +56,10 @@
  * @author Yegor Bugayenko (yegor@tpc2.com)
  * @version $Id$
  * @since 1.30
+ * @todo #561 When all the puzzles from
```

Comment:
actually, https://github.com/jcabi/jcabi-github/issues/923 is already implemented